src/engraving/libmscore/measurerepeat.cpp

```diff
@@ -63,15 +63,17 @@ Measure* MeasureRepeat::firstMeasureOfGroup() const
     return measure()->firstOfMeasureRepeatGroup(staffIdx());
 }
 
-const Measure* MeasureRepeat::referringMeasure() const
+const Measure* MeasureRepeat::referringMeasure(const Measure* measure) const
 {
-    Measure* firstMeasureRepeat = firstMeasureOfGroup();
+    Measure* referringMeasure = firstMeasureOfGroup()->prevMeasure();
 
-    if (!firstMeasureRepeat) {
```

Comment:
You should check if the result of firstMeasureOfGroup() is null (as it was before)